website/contributing/release-troubleshooting.md

```diff
@@ -42,3 +42,50 @@ git pull <remote> <branch> --tags
 # Make sure you push any local tags you added/removed
 git push <remote> <branch> --follow-tags
 ```
+
+### The Tag Has Been Pushed but NPM Failed to Publish React Native
+
+The release process takes various steps, but they can be summarised with these two:
+
+1. Create a version tag on the GitHub repository
+2. Publish all the artifacts on Maven and NPM
+
+Step 2 takes some time because it has to build everything we need to publish and it can happen that some builds fail or there are other issues.
+
+In case Step 1 completes successfully but step 2 fails, the repo is in a dirty situation and you need to address that.
+If you end up in this situation, follow these steps:
+
+1. Remove the tag that has been pushed (update the `X`, `Y` and `k` values appropriately):
+
+```sh
+git tag -d v0.X.Y[-RC.k]
+git push origin :v0.X.Y[-RC.k]
+```
+
+1. Revert the commit pushed by Step 1. The script we use for the release updates some versions in the `package.json`. The commit usually has this title `[0.X.Y-rc.k] Bump version numbers` for RCs or `[0.X.Y] Bump version numbers` for stable releases.
+
+```sh
+git revert <sha of the commit>
+git push origin 0.X-stable
+```
+
+1. Apply all the changes to fix the broken release. The remedies for this can could be of two types:
+   - you forgot to cherry-pick some commits. In this case, you follow the standard cherry-pick procedure for RCs and patches.
+   - you need to fix some bugs in the code. See the ["Add a local commit"](#add-a-local-commit) section below.
+1. Restart the release process **with the same tag**
```

Comment:
```suggestion
1. Once you have pushed all the necessary commits in the branch, re-trigger the release process **with the same tag** via the usual bump-oss script
```

Reply:
I tried not to explicitly mention the `bump-oss` script to be future-proof. Imagine that in a year we remove that script for something else, we would have to update this doc page as well.